src/components/ProfileDropdown.jsx

```diff
@@ -0,0 +1,122 @@
+import { withStyles, makeStyles } from '@material-ui/core/styles';
+import {
+  Avatar,
+  Container,
+  Divider,
+  Grid,
+  Typography,
+} from '@material-ui/core';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import NewIcon from '@material-ui/icons/NewReleases';
+import BookmarkIcon from '@material-ui/icons/Bookmark';
+import DoneIcon from '@material-ui/icons/Done';
+import ExitToAppIcon from '@material-ui/icons/ExitToApp';
+
+// hacky way to override Menu style
+const StyledMenu = withStyles({
+  paper: {
+    border: '1px solid black',
+    width: '300px',
+    background: '#f2f5fa',
+  },
+})(Menu);
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    marginTop: '10px',
+    marginBottom: '15px',
+  },
+  menuList: {
+    marginTop: '10px',
+    marginBottom: '10px',
+  },
+  profilePic: {
+    alignSelf: 'center',
+  },
+  medium: {
+    width: theme.spacing(6),
+    height: theme.spacing(6),
+  },
+  large: {
+    width: theme.spacing(8),
+    height: theme.spacing(8),
+  },
+}));
+
+export default function ProfileDropdown(props) {
+  const classes = useStyles();
+  const { signOut, name, email, photo } = props;
+  const [dropMenu, setDropMenu] = React.useState(null);
+
+  const handleClick = (event) => {
+    setDropMenu(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setDropMenu(null);
+  };
+
+  return (
+    <Container>
+      <Avatar src={photo} onClick={handleClick} className={classes.large} />
+      <StyledMenu
+        keepMounted
+        anchorEl={dropMenu}
+        open={Boolean(dropMenu)}
+        onClose={handleClose}>
+        <Grid container spacing={2} className={classes.root}>
+          <Grid item xs={3} sm={3} className={classes.profilePic}>
+            <Avatar src={photo} className={classes.medium} />
+          </Grid>
+          <Grid item xs={9} sm={9}>
+            <Typography variant="h6" component="h4">
+              <b>{name}</b>
+            </Typography>
+            <Typography component="h6" gutterBottom>
+              {email}
+            </Typography>
+            <a href="/home">Manage Your Profile</a>
+          </Grid>
+        </Grid>
+        <Divider />
+        <MenuItem className={classes.menuList}>
+          <ListItemIcon>
+            <NewIcon fontSize="medium" />
+          </ListItemIcon>
+          <ListItemText primary="New" />
+        </MenuItem>
+        <MenuItem className={classes.menuList}>
+          <ListItemIcon>
+            <BookmarkIcon fontSize="medium" />
+          </ListItemIcon>
+          <ListItemText primary="Saved" />
+        </MenuItem>
+        <MenuItem className={classes.menuList}>
+          <ListItemIcon>
+            <DoneIcon fontSize="medium" />
+          </ListItemIcon>
+          <ListItemText primary="Applied" />
+        </MenuItem>
```

Comment:
Lots of redundancy here. I suggest reading [List and Keys](https://reactjs.org/docs/lists-and-keys.html) and figuring out how to refactor this.

Reply:
Ok again same i'll read through this and include the changes in my next commit

Reply:
Alright I think I was able to refactor this by creating a function that basically creates `MenuItem` using props that are passed it should be on the most recent commit that I did